File: folders/python/javatpoint/factorial.py
n=int(input("enter any number:"))
factorial=1
if n<0:
    print("number can not be negative")
elif n==0:
    print("the factorial of 0 is 1")
else:
    for i in range(1,n+1):
        factorial*=i
    print("the factorial of" , n ,"is" ,factorial, "." )

File: folders/java/Factorial.java
import java.util.*;
public class Factorial
{
    public static void main(String args[])
    {
        Scanner s=new Scanner(System.in);
        System.out.println("Enter any number:");
        int n=s.nextInt();
        int fact=1;
        if(n<0)
        {
            System.out.println("enter a positive integer");
        }
        else if (n==0)
        {
            System.out.println("the factorial of 0 is 1");
        }
        else if (n>0){
        for (int i=1;i<=n;i++)
        {
            fact=fact*i;
        }
        System.out.println("the factorial of " + n + " is " + fact);
    }
    }
}
